sdk/eventhub/azure-eventhub/tests/livetest/synctests/test_eventprocessor.py

```diff
@@ -522,6 +522,8 @@ def claim_ownership(self, ownsership):
         (['ownership_expired', 'ownership_released'], ["0", "1", "2", "3"], 4),
         (['ownership_active0', 'ownership_active1'], ["0", "1"], 0),
         (['ownership_active0', 'ownership_self_owned'], ["0", "1"], 1),
+        (['ownership_active0', 'ownership_active1'], [str(i) for i in range(32)], 11),
```

Comment:
dumb question regarding to our greedy load balance algorithm:
why is this 11 (I guess it‘s 32/3 round up 11 ) not 16 as there are only two active owners?

Reply:
answered by yijun, it's two active ownership, so when running the code, there're actually three ownerships, the max expected per owner is 11